session-v7.go

```diff
@@ -32,14 +32,14 @@ import (
 	"github.com/minio/minio/pkg/quick"
 )
 
-func migrateSessionV5ToV6() {
+func migrateSessionV6ToV7() {
```

Comment:
Another thing is we cannot just do migrate and forget older v6 versions, we need to add migration for v5-->v6 and then v6-->v7 - just like how we do for config files.